custom/litmus-helm-agent/pkg/k8s/client.go

```diff
@@ -0,0 +1,82 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	corev1r "k8s.io/api/core/v1"
+
+	metav1r "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	// "k8s.io/client-go/tools/clientcmd"
+)
+
+func connectKubeApi() *kubernetes.Clientset {
+	config, err := rest.InClusterConfig()
```

Comment:
Should we also add support for outCluster config, for reference - https://github.com/litmuschaos/litmus/blob/643f3bffb87a20fee209a77b842ad7eda01d1e54/litmus-portal/graphql-server/pkg/k8s/client.go#L13

With outCluster config, it can be used locally for testing by go run .. --kubeconfig="path_to_kubeconfig"